### DISCUSSION TITLE
enable biometric or pin device lock support

### DISCUSSION THREAD

**Discussion by myusrn** [LSPosed/MagiskOnWSALocal#172] [Q&A] [answered]
This wsa with magisk and mindthegapps support to enable use of play store is excellent. I was trying to setup and run my authenticator apps for access on my desktop, vs having to pickup my phone, and found that it needs android device biometric or pin lock setting enabled. Is there a way to do that in this wsa installation?

**Comment by Howard20181** [comment, marked as answer]
Screen lock cannot be supported, the WSA display is special and once the lock is enabled you will not be able to unlock it, meaning you need to clear the data to access the system.

**Reply by arifszn** [reply]
I am currently locked out. Any idea how to clear the data?

**Comment by myusrn** [comment]
Any talk you've seen about eventually adding a story for wsa to provide pin lock support to enable use of apps that allow you to pull in credentials from authenticator and/or browser password stores?
